dictionaries/en_AU/src/additional_words.txt

```diff
@@ -1,30 +1,16 @@
-# Add words below
-# Consider adding words to ./shared-additional-words.txt if they are used by all flavours of English.
+# The CSpell English dictionary uses the SCOWL (Spell Checker Oriented Word Lists) dictionary as a
+# base: http://wordlist.aspell.net/
+#
+# Additionally, CSpell adds some extra words to that dictionary that are very common.
+#
+# There should be one entry per line.
+# Consider adding words to [shared-additional-words.txt](https://github.com/streetsidesoftware/cspell-dicts/blob/main/dictionaries/en_shared/src/shared-additional-words.txt)
+# if they are used by all flavors of English.
+#
 # cspell-tools: split
-Alpha Centauri
-amphitheatre
-Aruban
-Arubans
-bicep
-Christoph
 computerised
```

Comment:
I'm ok if you would like to add a shared: `shared-additional-words-ise.txt` file to include the `-ise` versions of words.

Reply:
I opened https://github.com/streetsidesoftware/cspell-dicts/pull/4397 to limit the scope of the PR, and to avoid fighting two dragons at the same time

- https://github.com/streetsidesoftware/cspell-dicts/pull/4397